src/LicenseFileWriter.ts

```diff
@@ -16,8 +16,9 @@ export default class LicenseFileWriter {
     options: IPluginOptions
   ): Promise<void> {
     const moduleDirs = this.getModuleDirs(filenames)
+    const includePackages = await options.includePackages();
     const licenseMeta = await this.licenseMetaAggregator.aggregateMeta(
-      moduleDirs
+      [...moduleDirs, ...includePackages]
```

Comment:
should we de-duplicate here to not read a folder/package.json twice?

Reply:
I was not sure if deduplication is necessary here. There seems to happen some deduplication already in aggregateMeta, because when I tried the e2e test with
```typescript
includePackages: () => [
    resolve(__dirname, 'example/node_modules/react')
  ]
```
the resulting json contained the react depdencency only once. But if we want to avoid the duplicate read we can also deduplicate here:

```typescript
[...new Set([...moduleDirs, ...includePackages])]
```

As for the issue that two packages `folderA/node_modules/some-package` and `folderB/node_modules/some-package` might have the same name and version but different licenses: I think that's very unlikely. Npm guarantees that the name and version combination is unique and does not allow to replace an already published version. The only way this could happen is if one of the packages is a local package or from a private registry and someone made the mistake of using the same name/version as an already existing package for his local/private package.

Reply:
I agree that this is unlikely and would be fine with listing such a package twice if this happens. I like the deduplication via `Set`!